imports/plugins/core/core/server/startup/startNodeApp.js

```diff
@@ -48,20 +39,22 @@ export default async function startNodeApp() {
       createUser(options) {
         return Accounts.createUser(options);
       },
-      mutations,
-      queries: finalQueries,
+      queries: coreQueries,
       rootUrl: ROOT_URL
     },
-    functionsByType,
     graphQL: {
       graphiql: Meteor.isDevelopment,
-      resolvers: finalResolvers,
-      schemas: [...coreSchemas, ...schemas]
+      resolvers: coreResolvers,
```

Comment:
I changed these fields a bit in a PR that was just merged, I can try to help resolve conflicts if they come up: https://github.com/reactioncommerce/reaction/pull/5145/files#diff-aec11b46e8437e7234ae8e5cc0c1a115